docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  backend:
+    restart: always
+    command: bash -c "cd lead && python manage.py runserver 0.0.0.0:8000"
+    build:  
+      context: ./moral_kombat_backend/
+      dockerfile: Dockerfile
+    volumes: 
+      - .:/app/code/
+    ports:
+      - "8000:8000"
+
+  editor-frontend:
+    restart: always
+    command : npm start
+    container_name: front
+    build:
+      context: ./Ghost_in_the_Shell/client_folder_react_app/ethisim/
+      dockerfile: Dockerfile
+    ports:
```

Comment:
Not needed for this PR, but you should look into enabling hot reloading. In other words, when I start up the application using Docker and make a code change (usually in the front-end UI), the browser automatically updates with the change.